articles/purview/tutorial-using-python-sdk.md

```diff
@@ -0,0 +1,335 @@
+    ```
+4.	You can now instantiate the two clients:
+    ```python
+    def get_credentials():
+        credentials = ClientSecretCredential(client_id=client_id, client_secret=client_secret, tenant_id=tenant_id)
+        return credentials
+
+    def get_purview_client():
+        credentials = get_credentials()
+        client = PurviewScanningClient(endpoint=f"https://{reference_name_purview}.scan.purview.azure.com", credential=credentials, logging_enable=True)  
+        return client
+    
+    def get_catalog_client():
+        credentials = get_credentials()
+        client = PurviewCatalogClient(endpoint=f"https://{reference_name_purview}.purview.azure.com/", credential=credentials, logging_enable=True)
+        return client
+    ```
+ 
+You are now able to do several operations in a similar fashion.
+
+## Register a data source
+In this section, you register your Blob Storage. For the sake of this tutorial, you will register it in the root collection of Purview.
+1.	First you need to define the following information to be able to register the Blob storage programmatically:
+    ```python
+    storage_name = "<name of your Storage Account>"
+    storage_id = "<id of your Storage Account>"
+    rg_name = "<name of your resource group>"
+    rg_location = "<location of your resource group>"
+    reference_name_purview = "<name of your Purview account>"
+    ```
+
+
+2.	For both clients and depending on the operations, you might need to also provide an input body. This is the case for data source registration:
+    ```python
+    ds_name = "myds"
+    
+    body_input = {
+            "kind": "AzureStorage",
+            "properties": {
+                "endpoint": f"https://{storage_name}.blob.core.windows.net/",
+                "resourceGroup": rg_name,
+                "location": rg_location,
+                "resourceName": storage_name,
+                "resourceId": storage_id,
+                "collection": {
+                    "type": "CollectionReference",
+                    "referenceName": reference_name_purview #here we use the root collection
+                },
+                "dataUseGovernance": "Disabled"
+            }
+        }    
+    ```
+    Note that in the above snippet, the name of the collection is the same as the Purview account because we work with the root collection that has the same name. Feel free to pass the name of an other collection you might have created.
+
+3.	Register the DS
+    ```python
+
+    try:
+        client = get_purview_client()
+    except AzureError as e:
+        print(e)
+
+    try:
+        response = client.data_sources.create_or_update(ds_name, body=body_input)
+        print(response)
+        print(f"Data source {ds_name} successfully created or updated")
+    except HttpResponseError as e:
+        print(e)
+    ```
+
+    When the operation succeeds, you can see an enriched body response from the client.
+
+All the operations are done in the same fashion. 
```

Comment:
All what operations?

Reply:
I specified the operations in my latest commit, is it clearer?